sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/ScannerForMSBuildInstaller.java

```diff
@@ -28,20 +28,18 @@
 import com.sonar.orchestrator.version.Version;
 import java.io.File;
 import java.net.URL;
-
 import javax.annotation.Nullable;
-
 import org.apache.commons.io.FileUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * Installs a given version of Scanner for MSBuild. It finds the zip into maven repositories
+ * Installs a given version of Scanner for MSBuild. It finds the zip into local maven repository
  *
  * @since 3.13
  */
 public class ScannerForMSBuildInstaller {
-  public static final String DEFAULT_SCANNER_VERSION = "2.1";
+  public static final String DEFAULT_SCANNER_VERSION = "2.2.0.24";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this IP "2.2.0.24" address configurable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1313)